docs/stencil-docs/reference-docs/styles-and-properties.md

```diff
@@ -3,6 +3,10 @@ Use the following data types in the `schema.json` file to create configurable op
 
 You can use your widget template settings to define a data type in the schema_translations property of the widget template schema.
 
+<!-- theme: info -->
+> #### `force_reload`
+> Some data types require the `force_reload` field. `force_reload` occurs when Page Builder reloads when the setting defined by `schema.json` changes.
```

Comment:
Rearrange and unpack the second sentence to describe the `force_reload` event more clearly.  When a data type requires `force_reload`, I'm imagining you define `"force_reload": true` somewhere?